### PR TITLE
Jsundet hartford ct

### PR DESCRIPTION
**Pull request by jsundet**
- This is a bullet.
- [ ] This is a list item.

### CODE REVIEW

**Comment by RajeshGooda** [comment]
Hi jsundet !!

File: jsundet-HartfordCT.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# HartfordCT
+### places to visit
```
Comment (RajeshGooda) [inline jsundet-HartfordCT.md:2]: Only one place to visit in HartfordCT ;) ?

**Review by RajeshGooda — APPROVED** [review]
great job !!

File: jsundet-HartfordCT.md (hunk 1 above)
Comment (jsundet) [reply jsundet-HartfordCT.md:2]: I can't keep up...    there are other places...

Comment (RajeshGooda) [reply jsundet-HartfordCT.md:2]: good to know that there are many places to visit :)

Comment (RajeshGooda) [reply jsundet-HartfordCT.md:2]: looks none one is interested in reviewing my changes. and workflow doesnt allow me review me my own changes :(.

could you please do me a favour reviewing my changes.

**Pull request merged**